test/common/access_log/access_log_formatter_fuzz_test.cc

```diff
@@ -21,6 +22,6 @@ DEFINE_PROTO_FUZZER(const test::common::access_log::TestCase& input) {
     ENVOY_LOG_MISC(debug, "EnvoyException: {}", e.what());
   }
 }
-
+} // namespace
```

Comment:
I hope it's not too much of a pain but I do think it's a bit more readable to have the blank line after a namespace declaration and before the end-namespace brace.  Do you think you could add those back?

Reply:
Yeah sure, will do